File: crosh/colors/base16-raeon.js
// Base16 Raeon
// Scheme: stagfoo

var color_scheme = {
        'base00': '#050533',
        'base01': '#150A52',
        'base02': '#121252',
        'base03': '#222278',
        'base04': '#E8D4F9',
        'base05': '#EFDFFF',
        'base06': '#F7F0FF',
        'base07': '#ffffff',
        'base08': '#FF008A',
        'base09': '#FA6F0A',
        'base0A': '#FDCA21',
        'base0B': '#85CF42',
        'base0C': '#00FBF1',
        'base0D': '#4A67FF',
        'base0E': '#964AFF',
        'base0F': '#FF05FA',
};

term_.prefs_.set('background-color', color_scheme.base00);
term_.prefs_.set('foreground-color', color_scheme.base05);
term_.prefs_.set('cursor-color', "rgba(239, 223, 255, 0.5)");

term_.prefs_.set('color-palette-overrides', 
                        [color_scheme.base00,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base05,
                        color_scheme.base03,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base07,
                        color_scheme.base09,
                        color_scheme.base0F,
                        color_scheme.base01,
                        color_scheme.base02,
                        color_scheme.base04,
                        color_scheme.base06]);

File: pygments/pygments_base16/base16-raeon.py
from pygments.style import Style
from pygments.token import (
    Comment, Error, Keyword, Literal, Name, Number, Operator, String, Text
)


class Base16Style(Style):
    base00 = '#050533'
    base01 = '#150A52'
    base02 = '#121252'
    base03 = '#222278'
    base04 = '#E8D4F9'
    base05 = '#EFDFFF'
    base06 = '#F7F0FF'
    base07 = '#ffffff'
    base08 = '#FF008A'
    base09 = '#FA6F0A'
    base0a = '#FDCA21'
    base0b = '#85CF42'
    base0c = '#00FBF1'
    base0d = '#4A67FF'
    base0e = '#964AFF'
    base0f = '#FF05FA'

    default_style = ''

    background_color = base00
    highlight_color = base02

    styles = {
        Text: base05,
        Error: base08,  # .err

        Comment: base03,  # .c
        Comment.Preproc: base0f,  # .cp
        Comment.PreprocFile: base0b,  # .cpf

        Keyword: base0e,  # .k
        Keyword.Type: base08,  # .kt

        Name.Attribute: base0d,  # .na
        Name.Builtin: base0d,  # .nb
        Name.Builtin.Pseudo: base08,  # .bp
        Name.Class: base0d,  # .nc
        Name.Constant: base09,  # .no
        Name.Decorator: base09,  # .nd
        Name.Function: base0d,  # .nf
        Name.Namespace: base0d,  # .nn
        Name.Tag: base0e,  # .nt
        Name.Variable: base0d,  # .nv
        Name.Variable.Instance: base08,  # .vi

        Number: base09,  # .m

        Operator: base0c,  # .o
        Operator.Word: base0e,  # .ow

        Literal: base0b,  # .l

        String: base0b,  # .s
        String.Interpol: base0f,  # .si
        String.Regex: base0c,  # .sr
        String.Symbol: base09,  # .ss
    }


from string import capwords  # noqa: E402
Base16Style.__name__ = 'Base16{}Style'.format(
    capwords('raeon', '-').replace('-', '')
)
globals()[Base16Style.__name__] = globals()['Base16Style']
del globals()['Base16Style']
del capwords
